compareTree.py

```diff
@@ -1,211 +1,140 @@
-import os, glob, re, csv, datetime # Imports python modules
+import os
+import glob
+import re
+import csv
+import datetime  # Imports python modules
+import createTree
+from configparser import ConfigParser
 from datetime import tzinfo
 
-cwd = os.path.dirname(os.path.realpath(__file__)) # Grabs current working directory
-cwd = cwd + '\BattleTech_Data\StreamingAssets\data\**\**' # Appends data path plus wildcards to current directory
+file_type_map = {
```

Comment:
Brilliant. I didn't know about mapping this way. As I was writing a thousand if statements I knew there was a better way. My first thought was switch-case but doing expression searches in that didn't seem possible.

Reply:
I don't like this way much better, although it is a little nicer than a bunch of ifs. It would be nice if we could use the filename directly, instead of assuming all files within a certain directory structure are of a certain type. Doing it this way also makes it hard to map filetypes in the case where one directory (/maps/ is an example) contains multiple filetypes we must distinguish between.

Reply:
Yah this is the definitely the toughest nut to crack. When I was manually parsing the manifest last night putting this list together, I was able to definitely see patterns we can search for. Just as an example: .dds and .png files in "emblems/portraits" are always added twice to the manifest, once as a sprite and once as a texture2d.

We need to go through all the file types and extensions and paths and find the easily identifiable patterns. Like if there's a .dds inside of THIS folder it must be a Sprite, if it's a .png it must be a Texture2d.

My propsal would start by filtering the file extensions first (json, png, dds, .bin, etc) THEN start filtering those based on the folders they reside in. LayerData.bin and similar files that end with a word will be easier to pick from those list to categorize them accordingly.